doc/source/io.rst

```diff
@@ -921,7 +921,7 @@ Note that performance-wise, you should try these methods of parsing dates in ord
 1. Try to infer the format using ``infer_datetime_format=True`` (see section below).
 
 2. If you know the format, use ``pd.to_datetime()``:
-   ``date_parser=lambda x: pd.to_datetime(x, format=...)``.
+   ``date_parser = lambda x: pd.to_datetime(x, format=...)``.
```

Comment:
This one is not necessarily correct, as `date_parser` is a keyword, and for keywords in function calls no spaces ...